File: test/EntityFramework/FunctionalTests/TestModels/TemplateModels/CsMonsterModel/ExternalTypes/AuditInfoMm.cs
namespace Another.Place
{
    using System;
    using FunctionalTests.ProductivityApi.TemplateModels.CsMonsterModel;

    public partial class AuditInfoMm
    {
        public AuditInfoMm()
        {
            Concurrency = new ConcurrencyInfoMm();
        }

        public DateTime ModifiedDate { get; set; }
        public string ModifiedBy { get; set; }

        public ConcurrencyInfoMm Concurrency { get; set; }
    }
}

File: test/EntityFramework/VBTests/VbMonsterModel/ExternalTypes/AuditInfoMm.vb
' Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
Imports System

Namespace Another.Place
    Partial Public Class AuditInfoMm
        Public Property ModifiedDate As Date
        Public Property ModifiedBy As String

        Public Property Concurrency As ConcurrencyInfoMm = New ConcurrencyInfoMm

    End Class
End Namespace
